src/sentry_config.py

```diff
@@ -0,0 +1,47 @@
+import base64
+from functools import lru_cache
+from configparser import ConfigParser
+
+import requests
+
+CACHE = {}
+SENTRY_CONFIG_API_URL = (
+    "https://api.github.com/repos/{owner}/.sentry/contents/sentry_config.ini"
+)
+
+# Remember the last 1000 orgs
+@lru_cache(maxsize=1000)
+def _org_dsn(org):
+    return CACHE[org]
```

Comment:
oh there's no need to do both the `CACHE` and `lru_cache` -- I would just do the latter

the only problem with this is it never expires :thinking: might make sense to do a time-expiry cache?

Reply:
The reason I'm using both and not just `lru_cache` is because the token is different in every call and since I need both value, I need to hide the token from the signature.

We've decided to remove caching for now since we need to add support for expiring it.